packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -128,6 +128,22 @@ const ruleset: any = {
         },
       },
     },
+    // RPC Code: RPC-Common-V1-05
+    LroErrorContent: {
+      description: "Error response content of long running operations must follow the error schema provided in the common types.",
+      message: "{{description}}",
+      severity: "error",
+      resolved: false,
+      formats: [oas2],
+      given:
+        "$[paths,'x-ms-paths'].*.*[?(@property === 'x-ms-long-running-operation' && @ === true)]^.responses[?(@property === 'default' || @property.startsWith('5') || @property.startsWith('4'))].schema.$ref",
+      then: {
+        function: pattern,
+        functionOptions: {
+          match: ".*/common-types/resource-management/v[0-9]+/types.json#/definitions/ErrorResponse",
```

Comment:
I think the v1/types.json is not allowed, as its schema doesn't meet the ARM RPC

Reply:
Updated it to support v2+